data/aerial-photography/index.html

```diff
@@ -104,7 +104,8 @@ <h3 class="text-left">Aerial Photography Layers for Utah</h3>
 <span><i class="fa fa-download"></i> <a href="{{ "/data/aerial-photography/mid-1990s-1-meter-black-white-orthophotography/" | prepend: site.baseurl }}">Mid 1990's 1 Meter Orthophotography</a></span><br />
 <span><i class="fa fa-download"></i> <a href="{{ "/data/aerial-photography/2003-1-meter-color-doqs-beaver-iron-only/" | prepend: site.baseurl }}">2003 1 Meter Color DOQs (Beaver and Iron Co Only)</a></span><br />
 <span><i class="fa fa-download"></i> <a href="{{ "/data/aerial-photography/1977-1-meter-black-white-doqs-salt-lake-county-only/" | prepend: site.baseurl }}">1977 1 Meter DOQs (Salt Lake County Only)</a></span><br />
-<span><i class="fa fa-download"></i> <a href="ftp://ftp.agrc.utah.gov/Imagery/UGS_SCS_Historical/">1936-1952 Soil Conservation Service (SCS) photomaps</a></span></p>
+<span><i class="fa fa-download"></i> <a href=""{{ "/data/aerial-photography/1936-1952-soil-conservation-service-scs-photomaps/" | prepend: site.baseurl }}">1936-1952 Soil Conservation Service (SCS) photomaps</a></span></p>
+<span><i class="fa fa-download"></i> <a href=""{{ "/data/aerial-photography/1970s-1-meter-blm-orthophotography/" | prepend: site.baseurl }}">1970s 1 Meter Color and Infrared DOQs</a></span></p>
```

Comment:
I made the changes to my master branch and there are a couple of commits
after that

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

On Wed, Jun 22, 2016 at 8:48 AM, steveoh <notifications@github.com> wrote:

> In data/aerial-photography/index.html
> <https://github.com/agrc/agrc.github.io/pull/163#discussion_r68066731>:
>
> > @@ -104,7 +104,8 @@ <h3 class="text-left">Aerial Photography Layers for Utah</h3>
> >  <span><i class="fa fa-download"></i> <a href="{{ "/data/aerial-photography/mid-1990s-1-meter-black-white-orthophotography/" | prepend: site.baseurl }}">Mid 1990's 1 Meter Orthophotography</a></span><br />
> >  <span><i class="fa fa-download"></i> <a href="{{ "/data/aerial-photography/2003-1-meter-color-doqs-beaver-iron-only/" | prepend: site.baseurl }}">2003 1 Meter Color DOQs (Beaver and Iron Co Only)</a></span><br />
> >  <span><i class="fa fa-download"></i> <a href="{{ "/data/aerial-photography/1977-1-meter-black-white-doqs-salt-lake-county-only/" | prepend: site.baseurl }}">1977 1 Meter DOQs (Salt Lake County Only)</a></span><br />
> > -<span><i class="fa fa-download"></i> <a href="ftp://ftp.agrc.utah.gov/Imagery/UGS_SCS_Historical/">1936-1952 Soil Conservation Service (SCS) photomaps</a></span></p>
> > +<span><i class="fa fa-download"></i> <a href=""{{ "/data/aerial-photography/1936-1952-soil-conservation-service-scs-photomaps/" | prepend: site.baseurl }}">1936-1952 Soil Conservation Service (SCS) photomaps</a></span></p>
> > +<span><i class="fa fa-download"></i> <a href=""{{ "/data/aerial-photography/1970s-1-meter-blm-orthophotography/" | prepend: site.baseurl }}">1970s 1 Meter Color and Infrared DOQs</a></span></p>
>
> we dealt with this already. this is the bad links with the multiple quotes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/agrc/agrc.github.io/pull/163/files/c145d48f48d71749ecb2e67cb3bd73b194b4ec5f..b1c2dcfd36e3ccbba181e3f231a7c618b89888f1#r68066731>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF6ebW6yBwFFcJByW5TCxCfvvu_q9ZUFks5qOUszgaJpZM4I72_C>
> .
>